unit/BasicRobolectricSample/app/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion rootProject.buildToolsVersion
+    defaultConfig {
+        applicationId "com.example.android.testing.robolectric.BasicRobolectricSample"
+        minSdkVersion 14
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+    }
+    lintOptions {
+        abortOnError false
+    }
+    testOptions {
```

Comment:
Please add a comment to indicate that this is required in order for Robolectric to work

Reply:
Done